tests/testthat/test-population_scaling.R

```diff
@@ -207,7 +207,63 @@ test_that("Postprocessing to get cases from case rate", {
 
 
 test_that("test joining by default columns", {
-  skip()
+  jhu <- case_death_rate_subset %>%
+    dplyr::filter(time_value > "2021-11-01", geo_value %in% c("ca", "ny")) %>%
+    dplyr::select(geo_value, time_value, case_rate)
+
+  reverse_pop_data <- data.frame(
+    geo_value = c("ca", "ny"),
+    values = c(1 / 20000, 1 / 30000)
+  )
+
+  r <- epi_recipe(jhu) %>%
+    step_population_scaling(case_rate,
+      df = reverse_pop_data,
+      df_pop_col = "values",
+      by = NULL,
+      suffix = "_scaled"
+    ) %>%
+    step_epi_lag(case_rate_scaled, lag = c(0, 7, 14)) %>% # cases
+    step_epi_ahead(case_rate_scaled, ahead = 7, role = "outcome") %>% # cases
+    recipes::step_naomit(recipes::all_predictors()) %>%
+    recipes::step_naomit(recipes::all_outcomes(), skip = TRUE)
+
+  expect_snapshot(prep <- prep(r, jhu))
+
+  expect_snapshot(b <- bake(prep, jhu))
```

Comment:
So these are the tests generating the snapshots. Looking at my local timemachine, this was a set of tests that was previously being skipped, and I added these as snapshot tests. I guess I'll just put this back to skipped for now.

Reply:
But yes, it does look like there are some lingering `Joining with `by = join_by(geo_value)` messages